### DISCUSSION TITLE
Problems after upgrading from 2.5.3 to 3.x

### DISCUSSION THREAD

**Discussion by hansgv** [amnuts/opcache-gui#71] [Q&A] [answered]
I'm experiencing two issues (PHP 7.4.16):

1. Page load time is huge. Typically 10-25 seconds with 3.0.0. In comparison, with 2.5.3 the page loads in 2-3 seconds.
2. My URL is http://my.domain.intra/opcache.php (internal domain available over VPN only). The external assets fail to load in versions above 3.0.0. In version 3.0.0 they load over https but in 3.0.1 and above it tries to load them over http which appears to fail for some reason (I can load the http URLs in command line of the server, they redirect to https)

No errors in the php-fpm error log.

Any thoughts on what I should do?

**Comment by amnuts** [comment, marked as answer]
Sorry you're having problems with the loading, @hansgv .  I honestly have to say that I find it much faster now, so not sure what's going on with your setup.  However, the two problems may be related - if it's having problems loading the js and css then it's going to feel like it's taking a long time to load the interface.

The first thing I'd recommend trying if changing the protocol of the js/css files.  As they stand it doesn't include the protocols so the browser will use http if you're on http and https otherwise.  But it sounds like you may need to force that.  So with the lines https://github.com/amnuts/opcache-gui/blob/master/index.php#L393-L395 (or https://github.com/amnuts/opcache-gui/blob/master/build/template.phps#L69-L71 and then run the rebuild script) change the `src` so that it reads `src="https://`.

Once you've done that and the external assets load, see how the speed feels.  If it's still sluggish you'll have to provide more info, I think.  Like how many objects do you have cached?  What opcache settings do you have?

**Reply by hansgv** [reply]
That did the trick. Thanks for the support!
